config/rbac/role.yaml

```diff
@@ -32,6 +32,7 @@ rules:
   - namespaces
   verbs:
   - list
+  - update
```

Comment:
can we get these changes in the helm chart too?

Reply:
yeah I plan to create a separate PR to update the helm chart